docs/integrations/sources/safetyculture.md

```diff
@@ -0,0 +1,57 @@
+# SafetyCulture
+
+This is the setup guide for the Safetyculture source connector which ingests data from the Safetyculture API.
+
+## Prerequisites
+
+This source uses the Authorization Bearer Token for handling requests. In order to obtain the credientials, you must first create a Safetyculture account.
+The API usage is only availabe for paid plans https://www.Safetyculture.com/
+
+You can create an API token under Account Settings -> Integrations -> API Tokens
+You can find more about their API here https://www.Safetyculture.com/api-reference/
+
+## Set up the Adjust source connector
+
+1. Click **Sources** and then click **+ New source**.
+2. On the Set up the source page, select **Safetyculture** from the Source type dropdown.
+3. Enter a name for your new source.
+4. For **API Token**, enter your API token obtained in the previous step.
+5. Click **Set up source**.
+
+## Supported sync modes
+
+The source connector supports the following [sync modes](https://docs.airbyte.com/cloud/core-concepts#connection-sync-modes):
+
+- Full Refresh
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| feed_users | id | DefaultPaginator | ✅ |  ❌  |
+| groups | id | No pagination | ✅ |  ❌  |
+| connections | id | No pagination | ✅ |  ❌  |
+| heads_up | id | DefaultPaginator | ✅ |  ❌  |
+| assets | id | DefaultPaginator | ✅ |  ❌  |
+| folders | id | DefaultPaginator | ✅ |  ❌  |
+| global_response_sets | responseset_id | No pagination | ✅ |  ❌  |
+| schedule_items | id | DefaultPaginator | ✅ |  ❌  |
+| actions |  | DefaultPaginator | ✅ |  ❌  |
```

Comment:
No primary key on stream, bad.

Reply:
One possible way is to add a transformation to nest out the task_id. Is that approach alright?

Reply:
Sure